thrift/src/main/thrift/atoms/explainer.thrift

```diff
@@ -2,16 +2,59 @@ namespace * contentatom.explainer
 namespace java com.gu.contentatom.thrift.atom.explainer
 #@namespace scala com.gu.contentatom.thrift.atom.explainer
 
+include "../shared.thrift"
+
 enum DisplayType {
   FLAT = 0,
   EXPANDABLE = 1,
   CAROUSEL = 2
 }
 
 struct ExplainerAtom {
+  1: optional string label
   /* the unique ID will be stored in the `atom` data*/
   2: required string title
```

Comment:
A title field also exists on the surrounding atom model so I'm not sure this is needed. See https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/contentatom.thrift#L76